app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/EventFormFragment.java

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.sweng.eventmanager.ui.eventShowcase.fragments;
+
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.content.IntentSender;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import ch.epfl.sweng.eventmanager.R;
+
+public class EventFormFragment extends AbstractShowcaseFragment {
+
+    String email;
+
+    private final int REQUEST_CODE = 1;
+
+    @BindView(R.id.contact_form_send_button)
+    Button send_button;
```

Comment:
Please respect java naming conventions!

```suggestion
    Button sendButton;
```